workflow/rules/filtering.smk

```diff
@@ -8,9 +8,9 @@ rule filter_by_annotation:
     params:
         filter=lambda w: config["calling"]["filter"][w.filter]
     conda:
-        "../envs/vep.yaml"
+        "../envs/vembrane.yaml"
     shell:
-        "(bcftools view {input} | filter_vep --filter \"{params.filter}\" --vcf_info_field ANN --only_matched | bcftools view -Ob > {output}) 2> {log}"
+        "vembrane {input} \"{params.filter}\" --output-fmt bcf --output {output} &> {log}"
```

Comment:
the syntax has changed, its now 
vembrane expression input

Reply:
Thanks. Changed that together with preprocessing the input by splitting multi allelic records.